internal/backends/qt/qt_window.rs

```diff
@@ -1714,6 +1714,37 @@ impl PlatformWindow for QtWindow {
     fn as_any(&self) -> &dyn std::any::Any {
         self
     }
+
+    fn handle_focus_change(&self, old: Option<ItemRc>, new: Option<ItemRc>) {
+        println!("QtWindow: Focus changed from {old:?} to {new:?}.");
+        let widget_ptr = self.widget_ptr();
+        if let Some(ai) = accessible_item(new) {
+            let item = &ai;
+            cpp! {unsafe [widget_ptr as "QWidget*", item as "void*"] {
+                auto accessible = QAccessible::queryAccessibleInterface(widget_ptr);
+                if (auto slint_accessible = dynamic_cast<Slint_accessible*>(accessible)) {
+                    slint_accessible->focusItem(item);
+                } else {
+                    qDebug() << "Accessible was null or not a Slint_accessible!";
```

Comment:
Should that be a qWarning? Or qFatal?

Reply:
I just removed this... There is little a user can do with this and worst case the accessibility will not work.